src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
This one is tricky. @apetitpa and @jvasseur are right about using the official HTML type name ... but at the same time, Symfony Forms define a lot of field types that don't match the HTML specification. So maybe it's OK tu use the user-friendly TelephoneType instead of the cryptic TelType? Let's ask a real form expert: @HeahDude what do you think? Thanks!